src/vs/platform/environment/node/argv.ts

```diff
@@ -96,12 +96,12 @@ export const options: Option[] = [
 	{ id: 'trace-options', type: 'string' },
 	{ id: 'prof-code-loading', type: 'boolean' },
 	// {{SQL CARBON EDIT}}
-	{ id: 'database', type: 'string', alias: 'D' },
-	{ id: 'server', type: 'string', alias: 'S' },
-	{ id: 'user', type: 'string', alias: 'U' },
-	{ id: 'command', type: 'string', alias: 'c' },
-	{ id: 'aad', type: 'boolean' },
-	{ id: 'integrated', type: 'boolean', alias: 'E' },
+	{ id: 'command', type: 'string', alias: 'c', cat: 'o', args: 'command name', description: localize('commandParameter', 'Name of command to run') },
+	{ id: 'database', type: 'string', alias: 'D', cat: 'o', args: 'database', description: localize('databaseParameter', 'Database name') },
+	{ id: 'server', type: 'string', alias: 'S', cat: 'o', args: 'server', description: localize('serverParameter', 'Server name') },
+	{ id: 'user', type: 'string', alias: 'U', cat: 'o', args: 'user name', description: localize('userParameter', 'User name for server connection') },
```

Comment:
can you actually use space in the args, i noticed a few of the arguments have space in it. e.g. command name.

Reply:
looks like the convention is to use a hyphen
````
                                                  --list-extension.
  --install-extension <extension-id | path-to-vsix> Installs or updates the
                                                    extension. Use `--force`
                                                    argument to avoid
                                                    prompts.

Reply:
````
Options
  -d --diff <file> <file>           Compare two files with each other.
  -a --add <folder>                 Add folder(s) to the last active window.
  -g --goto <file:line[:character]> Open a file at the path on the specified
                                    line and character position.
  -n --new-window                   Force to open a new window.
  -r --reuse-window                 Force to open a file or folder in an
                                    already opened window.
  -w --wait                         Wait for the files to be closed before
                                    returning.
  --locale <locale>                 The locale to use (e.g. en-US or zh-TW).
  --user-data-dir <dir>             Specifies the directory that user data is
                                    kept in. Can be used to open multiple
                                    distinct instances of Code.
  -v --version                      Print version.
  -h --help                         Print usage.
  --folder-uri <uri>                Opens a window with given folder uri(s)
  --file-uri <uri>                  Opens a window with given file uri(s)
  -c --command <command-name>       Name of command to run
  -D --database <database>          Database name
  -S --server <server>              Server name
  -U --user <user-name>             User name for server connection
  --aad                             Use Azure Active Directory authentication
                                    for server connection
  -E --integrated                   Use Integrated authentication for server
                                    connection